Readme.markdown

```diff
@@ -138,24 +138,16 @@ cabal configure --enable-tests && cabal build -j
 ```
 
 To execute the tests, you need a running PostgreSQL server with an empty
-database called _test_. Tests are executed through cabal as follows:
+database called _test_. You can start one using docker, e.g.:
 
 ```bash
-cabal configure --enable-tests && cabal test
+docker run --rm -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_DB=test postgres:15-alpine
```

Comment:
Should we also upgrade PostgreSQL in the CI file from 14 to 15 as well to stay consistent?